src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,135 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private final int maxSize = 10;
```

Comment:
кразе цю змінну константою зробити ( static final ) 
```suggestion
    private final int DEFAULT_SIZE = 10;
```
\+ так як в тебе є метод resize це не макс розмір

Reply:
Так и хотел сделать static final но почему то были ошибки при mvn clean package. Сейчас ок)